.github/workflows/master-workflow.yml

```diff
@@ -1,91 +1,71 @@
 name: Master workflow
 
-# 1. When a PR review is requested, run tests on that PR
-# 2. If all of the tests pass, allow the PR to be merged into `dev`
-# 3. Whenever a PR is merged to `dev`, bump version number in `dev`
-
 on:
   pull_request:
     branches:
-      - 'master-test'
-    paths-ignore:
-      - 'docs/**'
-      - 'aerospike-stubs/**'
-
-  # So we can test changes to the test-server-rc workflow
-  workflow_dispatch:
-    inputs:
-      run_server_release_tests:
-        description: 'DEBUG: Run server release tests in build-wheels workflow?'
-        type: boolean
-        default: false
+      - master-test
 
 jobs:
-  get-jfrog-stage-version:
-    #if: ${{ github.event_name == 'workflow_dispatch' || github.event.pull_request.merged == true }}
-    uses: ./.github/workflows/get-jfrog-stage-version.yml
+  master-get-jfrog-version:
+    uses: ./.github/workflows/master-get-jfrog-version.yml
     secrets: inherit
 
-  bump-dev-number:
-    uses: ./.github/workflows/bump-master-version.yml
+  bump-master-number:
+    uses: ./.github/workflows/master-bump-version.yml
     needs: [
-      get-jfrog-stage-version,
+      master-get-jfrog-version,
     ]
     secrets: inherit
 
   publish-to-npm:
     name: Publish to npm
     needs: [
-      get-jfrog-stage-version,
-      bump-dev-number,
+      master-get-jfrog-version,
+      bump-master-number,
     ]
-    uses: ./.github/workflows/upload-jfrog-build-to-npm.yml
+    uses: ./.github/workflows/master-upload-jfrog-build-to-npm.yml
     with:
-      old_version: ${{ needs.get-jfrog-stage-version.outputs.stage_version }}
-      #old_version: 6.2.1-dev.4
-      new_version: ${{ needs.bump-dev-number.outputs.new_master_version }}
-      #new_version: 6.2.1-dev.4
+      old_version: ${{ needs.master-get-jfrog-version.outputs.stage_version }}
+      new_version: ${{ needs.bump-master-number.outputs.new_master_version }}
     secrets: inherit
 
   verify-npm-install:
-    name: verify npm install works correctly
+    name: verify npm ci works correctly
```

Comment:
I think in this one instance we're actually verifying `npm install` of our own package, not `npm ci`. Right?